NEWS

```diff
@@ -1,3 +1,20 @@
+09-Aug-2018 IGNITION v0.27.0
+
+    Features:
+
+    - Ignition is now built as a Position Independent Executable (PIE)
+    - Blackbox tests now run against all versions (within the same major
+      version) after their minimum version
+    - Status reporting for Packet
+    - Files are now relabeled for SELinux after boot (requires setting
+      selinuxRelabel=true)
+
+    Bug Fixes:
+
+    - Directories that are specified in appended configs are now always created
+      second (allowing the appended config to control the permissions)
```

Comment:
`Directories specified in both base and appended configs are always created with the permissions specified in the appended config`?